Simplified/AxisService/NYPLAxisItemDownloader.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  NYPLAxisItemDownloader.swift
+//  Open eBooks
+//
+//  Created by Raman Singh on 2021-05-12.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+protocol NYPLAxisItemDownloadTerminationListening: class {
+  func downloaderDidTerminate()
+}
+
+protocol NYPLAxisItemDownloading: class {
+  var delegate: NYPLAxisItemDownloadTerminationListening? { get set }
+  var dispatchGroup: DispatchGroup { get }
+  var shouldContinue: Bool { get }
+  func downloadItem(from url: URL, at writeURL: URL)
+  func leaveGroupAndStopDownload()
+}
+
+class NYPLAxisItemDownloader: NYPLAxisItemDownloading {
+  
+  // Content Downloading constants
+  static private let writeFailureSummary = "AXIS: Failed writing item to specified write URL"
+  static private let failedDownloadingItemSummary = "AXIS: Failed downloading item"
```

Comment:
`writeFailureSummary` should follow the naming convention of this property

Reply:
Agreed. Fixed.